frame/nomination-pools/src/lib.rs

```diff
@@ -408,43 +407,40 @@ pub struct BondedPoolStorage<T: Config> {
 	pub delegator_counter: u32,
 	/// See [`BondedPool::depositor`].
 	pub depositor: T::AccountId,
-	/// See [`BondedPool::admin`].
+	/// See [`BondedPool::root`].
 	pub root: T::AccountId,
 	/// See [`BondedPool::nominator`].
 	pub nominator: T::AccountId,
 	/// See [`BondedPool::state_toggler`].
 	pub state_toggler: T::AccountId,
 }
 
+/// A wrapper for bonded pools, with utility functions.
+///
+/// The main purpose of this is to wrap a [`BondedPoolInner`], with the account id of the pool, for
+/// easier access.
 #[derive(RuntimeDebugNoBound)]
 #[cfg_attr(feature = "std", derive(Clone, PartialEq))]
 pub struct BondedPool<T: Config> {
-	/// Points of the pool.
-	///
-	/// Each delegator has some corresponding the points. The portion of points that belong to a
-	/// delegator represent the portion of the pools bonded funds belong to the delegator.
-	points: BalanceOf<T>,
-	/// State of the pool.
-	state: PoolState,
-	/// Count of delegators that belong to this pool.
-	delegator_counter: u32,
-	/// Account that puts down a deposit to create the pool.
-	///
-	/// This account acts a delegator, but can only unbond if no other delegators belong to the
-	/// pool.
-	depositor: T::AccountId,
-	/// Can perform the same actions as [`Self::nominator`] and [`Self::state_toggler`].
```

Comment:
Need to move the docs for these fields to inner, since there is no longer `BondedPool::root` etc